### PR TITLE
Add support for accented characters

### PR DESCRIPTION
**Pull request by StanBoyet**
Library from this SO answer : http://stackoverflow.com/a/9667817/2726340
I added a normalize() call to be sure about the encoding. On my mac, it appears that my filenames were in NFD, the library was in UTF8, so nothing was matched.

Now it seems to be working fine. Don't hesitate to make changements if necessary of course.

PR from issue #32

### CODE REVIEW

**Comment by seanadkinson** [comment]
Looks like there is a [npm module that does the same thing](https://www.npmjs.com/package/latinize), so maybe better to rely on that.  Also, I don't want to be changing the `String.prototype` for everyone without them asking, so that's another vote for the npm module.

File: s3upload.js [hunk 1]
```diff
@@ -60,7 +66,7 @@ S3Upload.prototype.createCORSRequest = function(method, url) {
 
 S3Upload.prototype.executeOnSignedUrl = function(file, callback) {
     var xhr = new XMLHttpRequest();
-    var fileName = file.name.replace(/\s+/g, "_");
+    var fileName = file.name.latinize();
```
Comment (seanadkinson) [inline s3upload.js]: I think we still want the whitespace to underscore conversion.  It doesn't look like `latinize()` will handle that, so can we add that back?

**Comment by seanadkinson** [comment]
Couple things to consider, but thanks for the PR :+1:

**Comment by StanBoyet** [comment]
Let me know if I can be of help. In the meantime, do you know how I could manage to do this thing ? Maybe by having a jquery listener before or something like this ?

**Comment by seanadkinson** [comment]
Can you add the dependency on the `latinize` library from npm, and add back the whitespace substitution?  I'll merge and push an update once this is ready.

**Comment by StanBoyet** [comment]
Working fine on my end :
- filename : "étage précédent.gif"
- Mac OSX 10.11.1
- browsers
  - Firefox 42
  - Chrome 46
  - Safari 9.0.1

**Comment by seanadkinson** [comment]
Looks great.  I'll bump the minor version number since this adds dependencies, and will push a release soon.

**Comment by seanadkinson** [comment]
Pushed `v1.2.0` with these changes.

**Pull request merged**